configuration/provider_aggregator.go

```diff
@@ -4,8 +4,10 @@ import (
 	"encoding/json"
 	"reflect"
 
+	"github.com/containous/traefik/acme"
 	"github.com/containous/traefik/log"
 	"github.com/containous/traefik/provider"
+	provideracme "github.com/containous/traefik/provider/acme"
```

Comment:
You used `acmeprovider` but [here](https://github.com/containous/traefik/pull/2889/files#diff-bb76aa29d7113847a8b689dc0e0b95ccR12) you used `providerACME`.
Could you rename one to be consistent please ?